### ISSUE TITLE
Chaos damage on hit from Despair is not included in any calculations

### ISSUE THREAD

**Issue by lolbifrons** [Openarl/PathOfBuilding#733]
As far as I can tell, the mod "Adds (9-46) to (12-57) Chaos Damage to Hits against Cursed Enemies" from the Despair curse isn't factored into any damage calculations.  I'm using SRS, but I also checked for my ball lightning, and there was no chaos damage component for that on the Calcs tab either.

**Comment by Openarl** [comment]
Indeed; I haven't quite figured out how to handle that particular stat yet. It is fairly high on the list of issues to look into once I have all the critical stuff out of the way (new uniques, etc).

**Comment by MalteMagnussen** [comment]
The issue seems to still exist.
